aea/connections/scaffold/connection.py

```diff
@@ -33,22 +31,15 @@ class MyScaffoldConnection(Connection):
 
     connection_id = PublicId.from_str("fetchai/scaffold:0.1.0")
 
-    def __init__(
-        self,
-        configuration: ConnectionConfig,
-        identity: Optional[Identity] = None,
-        crypto_store: Optional[CryptoStore] = None,
-    ):
+    def __init__(self, **kwargs):
         """
         Initialize a connection to an SDK or API.
 
         :param configuration: the connection configuration.
         :param crypto_store: object to access the connection crypto objects.
         :param identity: the identity object.
         """
-        super().__init__(  # pragma: no cover
-            configuration=configuration, crypto_store=crypto_store, identity=identity
-        )
+        super().__init__(**kwargs)  # pragma: no cover
```

Comment:
The reason we did miss this before is because we had no test. Pragma no cover needs removing and we need to test instantiation of an empty scaffolded class.

Reply:
I see test is there now, so remove pragma statement please

Reply:
the problem that file is not executed, but it's copy, this is why pragma: nocover is used